tests/test_utils.py

```diff
@@ -33,14 +33,21 @@ def test_pyproject_toml_matches_version():
     "test_input, expected",
     [
         ("abcde:12345", "12345"),
-        ("abcde:my-name", "my_name"),
+        ("abcde:my-name", None),
+        ("d347875:alpha_user_some_user", "some_user"),
```

Comment:
probably wouldn't hurt to have a test case that includes usernames with hyphens in too